docs/sdk/access-control/quick-start.md

```diff
@@ -476,4 +476,4 @@ By now you should have successfully created an Access Control Condition and perf
 3. [Off-Chain Conditions](../access-control/lit-action-conditions.md).
 4. [Custom Contract Calls](../access-control/evm/custom-contract-calls.md).
 
-Did you find this guide helpful? If not, please [get in touch](https://docs.google.com/forms/d/e/1FAIpQLScBVsg-NhdMIC1H1mozh2zaVX0V4WtmEPSPrtmqVtnj_3qqNw/viewform?usp=send_form).
\ No newline at end of file
+Did you find this guide helpful? If not, please [get in touch](https://docs.google.com/forms/d/e/1FAIpQLScBVsg-NhdMIC1H1mozh2zaVX0V4WtmEPSPrtmqVtnj_3qqNw/viewform?usp=send_form).
```

Comment:
Not sure what's going on with this line - didn't change anything here on purpose

Reply:
yeah not sure either.  it looks the same tho, prob fine to ignore.